scripts/commands.js

```
@@ -169,8 +169,91 @@ async function main() {
+        await setTestConfig()
+      } else if (args[1] == 'accounts') {
+        await checkAccounts()
+      } else if (args[1] == 'interactive' || args.length == 1) {
+        await interactiveConfig()
+      } else {
+        console.log('Config options: show, files, test, accounts, interactive')
+      }
+      break;
+
+    case 'check':
+      const checkSubCommand = process.argv[3];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)